packages/scalars/package.json

```diff
@@ -44,20 +44,20 @@
       "import": "./dist/es/src/index.js",
       "types": "./dist/types/src/index.d.ts"
     },
-    "./src/scalars/index": {
-      "require": "./dist/cjs/src/scalars/src/scalars/index.js",
-      "import": "./dist/es/src/scalars/src/scalars/index.js",
-      "types": "./dist/types/src/scalars/src/scalars/index.d.ts"
+    "./src\\scalars\\index": {
```

Comment:
This changes are probably unintentional, please check

Reply:
Indeed. Fixed